examples/inference-deployments/mpt/README.md

```diff
@@ -1,22 +1,57 @@
-## Model Description
+## Inference with MPT-7B 
```

Comment:
Can we please include a "setup" section that tells the user how to setup their environment and account to actually be able to run this?

Reply:
this could also be a pointer to a top level readme with these instructions

Reply:
Sounds good, I did add it at the top level draft that I have. Will edit!